index.md

```diff
@@ -298,7 +299,18 @@ world = 'World'
 a_string = '%s %s' % (hello, world)
 ```
 
-### 4.0 Styleguide
+### 4.0 HTML
+
+- double quotes for attribute values
```

Comment:
need a newline between this and the code snippet, markdown is picky, also this needs to be sentence case